calico/networking/configuring/vxlan-ipip.mdx

```diff
@@ -49,7 +49,7 @@ Encapsulation of workload traffic is typically required only when traffic crosse
 **Limitations**
 
 - IP in IP supports only IPv4 addresses
-- VXLAN in IPv6 is only supported for kernel versions >= 3.12
+- VXLAN in IPv6 is only supported for kernel versions >=4.19.1 or redhat kernel version >= 4.18.0
```

Comment:
I'm not sure about including specifics for redhat. How about every other kernel variation? @blue-troy in https://github.com/projectcalico/calico/pull/7505 you mentioned that https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?id=6a4aa53a94615ab4a8cf66b24e3780719378a059 fixes the issue you were having, right? That doesn't seem VXLAN specific, couldn't that bug happen regardless of encap? @mgleung and @caseydavenport, what's your opinion on mentioning the fix commit (or something to that effect), or mentioning (or not) a redhat kernel version?

Reply:
Year, then kernel include that commit fixs me issue in both redhat kernel and official kernel. I'm not currently using any other kernels such as Debian, and haven't tested them. I think the completeness of the documentation is quite important, and mentioning here may help others and avoid wasting several workdays for them. If other people have other kernel variations, we can continue to supplement them later. Having a comprehensive documentation is not a bad thing.